Assets/Branch/BranchAndroidWrapper.cs

```diff
@@ -348,17 +262,15 @@ public class BranchAndroidWrapper {
     
     private static AndroidJavaClass _getBranchClass() {
         if (_branchClass == null) {
+            // The looper needs to be prepared before calling methods on the Branch class apparently.
```

Comment:
This was removed because it broke other things. Please consider reverting, I'm not sure why you changed this back.